core/src/main/java/org/ehcache/core/Ehcache.java

```diff
@@ -1079,4 +1079,25 @@ public AtomicInteger getActualRemoveCount() {
     }
   }
 
+  @Override
+  public String toString() {
+    return "statusTransitioner:\n    "
```

Comment:
This (and the CM level) is way more than configuration - it even gives information on the current state. Is that really necessary?